### PR TITLE
(maint) Update honeysql (v2) to 2.3.911

### PR DESCRIPTION
**Pull request by austb**
Update honeysql v2 to latest. To upgrade PuppetDB from honeysql v1, we
need to use some of the postgres operators added in 2.2.891

### CODE REVIEW

**Comment by jonathannewman** [comment]
What is the ticket that is motivating this work?

**Comment by austb** [comment]
PDB-5519

**Comment by jonathannewman** [comment]
Thanks.  That ticket doesn't seem to be on a scrum board.  Unless this is a ticket driven by product and has clear customer value, making this change at this point is not a good idea: Any change we make has the potential to introduce risk, and this has the potential to introduce risk into multiple components, and we are less than two weeks from release with multiple changes going in around postgres, including new versions of postgres and drivers.

**Comment by austb** [comment]
Yeah, I wasn't on a scrum team until yesterday :)

This change is not required for this round of PE releases, so happy to wait until after this release.

**Comment by jonathannewman** [comment]
Perfect.  Thanks @austb

**Comment by austb** [comment]
Both release branches have been created, so I believe this is safe to land now.

**Comment by jonathannewman** [comment]
PE-34323 needs to be done before this can be merged so we understand any potential impact.

**Comment by austb** [comment]
Is there anything you need/want me to do? I'm happy to run both those test sets if Dumpling is wrapped up in other things.

**Comment by austb** [comment]
tests were successful on orcehstrator and pe-rbac-services `main` branches. Their 2019.8.x branches will be unaffected because they are on the clj-parent 4 branch

**Comment by jonathannewman** [comment]
Thanks @austb !

**Review by jonathannewman — APPROVED** [review]
LGTM!

**Pull request merged**
